DESCRIPTION

```diff
@@ -33,6 +33,7 @@ Imports:
    plotly
```

Comment:
`MsCoreUtils (>= 1.1.2)` should become `MsCoreUtils (>= 1.7.2)`, otherwise `QFeatures` installation fails with `Error: object ‘aggregate_by_matrix’ is not exported by 'namespace:MsCoreUtils'`

Reply:
Indeed, thank you - addressed now.